src/status_im/data_store/realm/messages.cljs

```diff
@@ -57,9 +58,22 @@
   [message-id]
   (realm/exists? @realm/account-realm :message {:message-id message-id}))
 
+(defn- prepare-for-transit 
+  "Following Transit documentation, escape leading ~'s
+   in content strings by prepending a ~.
+   This prepares for subsequent fetching from Realm where
+   Transit is used for JSON parsing."
+  [{:keys [content] :as message}]
+  (if (and (not (string/blank? content))
+           (string/starts-with? content "~"))
```

Comment:
Looks like `~` is not the only special character. Also we probably want to do this generically, not only for this particular filed?